extensions/amp-google-assistant-assistjs/0.1/libs/closure-bundle.js

```diff
@@ -0,0 +1,101 @@
+
+  /*
+
+ Copyright The Closure Library Authors.
+ SPDX-License-Identifier: Apache-2.0
+*/
+var k;function aa(a){var b=0;return function(){return b<a.length?{done:!1,value:a[b++]}:{done:!0}}}var ba="function"==typeof Object.defineProperties?Object.defineProperty:function(a,b,c){if(a==Array.prototype||a==Object.prototype)return a;a[b]=c.value;return a};
```

Comment:
This is a significant amount of code for the extension. Is all of this really necessary just to add event listeners to the iframes?

Reply:
Yes unfortunately these dependencies are necessary to use goog.messaging.* libraries. The entire assist.js channel communication is using goog.messaging.* libraries, including, PortChannel, RespondingChannel, PortOperator, etc.